rust/datafusion/src/sql/planner.rs

```diff
@@ -963,7 +983,12 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                             .collect::<Result<Vec<Expr>>>()?
                     };
 
+                    if let CaseStyle::LikePostgreSQL = case_style {
+                        input_name = input_name.to_uppercase();
```

Comment:
I think postgres does `to_lowercase` -- this also seems to be inconsistent with the clause above (that also uses `to_lowercase`.

Reply:
If we used to_LowerCase, many of the test cases would fail, such as the test case below:

#[tokio::test]
    async fn count_basic() -> Result<()> {
        let results = execute("SELECT COUNT(c1), COUNT(c2) FROM test", 1).await?;
        assert_eq!(results.len(), 1);

        let expected = vec![
            "+-----------+-----------+",
            "| COUNT(c1) | COUNT(c2) |",
            "+-----------+-----------+",
            "| 10        | 10        |",
            "+-----------+-----------+",
        ];
        assert_batches_sorted_eq!(expected, &results);
        Ok(())
    }

Reply:
@alamb 

I would like to put the modification of the test case in another PR, the current PR just like that, is that OK?